### ISSUE TITLE
Some nested objects are not showing

### ISSUE THREAD

**Issue by Sceat** [HydreIO/pino-human#3]
![image](https://user-images.githubusercontent.com/11330271/136964637-cea16892-e42e-4bb4-8c2c-7db34f8cdeb3.png)

**Comment by Sceat** [comment]
I think it has to do with arrays

**Comment by DeltaEvo** [comment]
Not really, this has to do with an option on inspect https://nodejs.org/api/util.html#utilinspectobject-showhidden-depth-colors, the depth is set to 2 inside index.js
We should maybe set it to Infinity or take the value from an env variable

**Comment by Sceat** [comment]
Infinity seems a great default here
